### DISCUSSION TITLE
What's the best way to learn what some of the new things actually are?

### DISCUSSION THREAD

**Discussion by vaderag** [emukidid/swiss-gc#579] [Q&A]
So, just updated to the new Swiss and all excellent, but I'm not quite sure what some of the new things are...

For example:
Use macro to change video width - what exactly does this do? If I'm forcing 576i does this do anything?
Add clean boot by pressing L+A - what's the purpose of this? 
Fix aspect ratio of Burnout - what was wrong before with these aspects?
Support memory card fast operation mode - do I want this?

I'm not expecting everything spelled out and I'm happy to read if they're outlined somewhere but struggling to find a way to understand the release notes better! Basically, I want to ensure I'm getting the most out of Swiss!

**Comment by Extrems** [comment]
> Use macro to change video width - what exactly does this do? If I'm forcing 576i does this do anything?

This is just me refactoring code I had just added.

> Add clean boot by pressing L+A - what's the purpose of this?

It's meant to cater to GC Loader users that don't want to deal with Swiss' nonsense. It's also necessary to boot things like the Aging Disc or the Service Disc.

> Fix aspect ratio of Burnout - what was wrong before with these aspects?

https://twitter.com/GigaBoots/status/1429383561581318145

It's an artifact of the ITU-R BT.601 video standard. Most games with an external resolution of 640x480 are affected.
Those using 640x448 are close enough (should be 656x448) and 666x448 Nintendo games are perfect.

> Support memory card fast operation mode - do I want this?

It reduces hypervisor overhead and gives a 1:1 relationship between memory card writes and SD card writes.
This is limited to supporting games released late in the GameCube's lifecycle, and is the same mode enabled by the Memory Card 1019.

**Reply by vaderag** [reply]
Thanks for the insight! Is there anywhere this kind of thing is mentioned/discussed for me to track down myself?

**Comment by vaderag** [comment]
@Extrems related to the above thread, wondering  what the reasoning for the change to IGR is in the latest release?
Need to retrain the kids now ;)

**Reply by Extrems** [reply]
See issue #593.
